osgp/platform/osgp-adapter-kafka-distributionautomation/src/main/resources/db/migration/V20210104135300503__Initial_version_PostgreSQL.sql

```diff
@@ -0,0 +1,51 @@
+DO
+$$
+BEGIN
+
+IF NOT EXISTS (
+    SELECT 1
+    FROM   pg_tables
+    WHERE  schemaname = current_schema
+    AND    tablename  = 'location') THEN
+
+    CREATE TABLE location
+	(
+	    id                        bigserial                   NOT NULL,
+	    creation_time             timestamp without time zone NOT NULL,
+	    modification_time         timestamp without time zone NOT NULL,
+	    version                   bigint,
+	    substation_identification character varying(12)       NOT NULL,
+	    name                      character varying(32)       NOT NULL,
+	    CONSTRAINT location_pkey PRIMARY KEY (id),
+	    CONSTRAINT location_substation_identification_key UNIQUE (substation_identification)
+	);
+
+    ALTER TABLE public.location OWNER TO osp_admin;
```

Comment:
Wouldn't it be better to leave the `public.` prefix out (here and with `public.feeder` below) just in case the SQL script would ever be used in another schema?